draft-ietf-taps-impl.md

```diff
@@ -289,31 +289,31 @@ Differences in possible protocol compositions and options can also provide a bra
 This approach is commonly used for connections with optional proxy server configurations. A single connection might have several options available: an HTTP-based proxy, a SOCKS-based proxy, or no proxy. As above, these options should be ranked based on preference, system policy, and performance and attempted in succession.
 
 ~~~~~~~~~~
-1 [www.example.com:80, Any, HTTP/TCP]
-  1.1 [192.0.2.8:80, Any, HTTP/HTTP Proxy/TCP]
-  1.2 [192.0.2.7:10234, Any, HTTP/SOCKS/TCP]
-  1.3 [www.example.com:80, Any, HTTP/TCP]
-    1.3.1 [192.0.2.1:80, Any, HTTP/TCP]
+1 [www.example.com:80, Wi-Fi, HTTP/TCP]
+  1.1 [192.0.2.8:80, Wi-Fi, HTTP/HTTP Proxy/TCP]
+  1.2 [192.0.2.7:10234, Wi-Fi, HTTP/SOCKS/TCP]
+  1.3 [www.example.com:80, Wi-Fi, HTTP/TCP]
+    1.3.1 [192.0.2.1:80, Wi-Fi, HTTP/TCP]
 ~~~~~~~~~~
 
 This approach also allows a client to attempt different sets of application and transport protocols that, when available, could provide preferable features. For example, the protocol options could involve QUIC {{I-D.ietf-quic-transport}} over UDP on one branch, and HTTP/2 {{!RFC7540}} over TLS over TCP on the other:
 
 ~~~~~~~~~~
-1 [www.example.com:443, Any, Any HTTP]
-  1.1 [www.example.com:443, Any, QUIC/UDP]
-    1.1.1 [192.0.2.1:443, Any, QUIC/UDP]
-  1.2 [www.example.com:443, Any, HTTP2/TLS/TCP]
-    1.2.1 [192.0.2.1:443, Any, HTTP2/TLS/TCP]
+1 [www.example.com:443, Wi-Fi, HTTP]
+  1.1 [www.example.com:443, Wi-Fi, HTTP3/QUIC/UDP]
+    1.1.1 [192.0.2.1:443, Wi-Fi, HTTP3/QUIC/UDP]
+  1.2 [www.example.com:443, Wi-Fi, HTTP2/TLS/TCP]
+    1.2.1 [192.0.2.1:443, Wi-Fi, HTTP2/TLS/TCP]
 ~~~~~~~~~~
 
 Another example is racing SCTP with TCP:
 
 ~~~~~~~~~~
-1 [www.example.com:80, Any, Any Stream]
-  1.1 [www.example.com:80, Any, SCTP]
-    1.1.1 [192.0.2.1:80, Any, SCTP]
-  1.2 [www.example.com:80, Any, TCP]
-    1.2.1 [192.0.2.1:80, Any, TCP]
+1 [www.example.com:80, Wi-Fi, Stream+Reliable]
```

Comment:
We have a transport property profile described in the API draft called reliable-inorder-stream. Should we use that rather than Stream+Reliable? Just a thought, I am also fine with Stream+Reliable.

Reply:
+1 to using "reliable-inorder-stream"

Reply:
done